doc/tor.1.txt

```diff
@@ -3044,6 +3044,32 @@ The next section describes the per service options that can only be set
    The HAProxy version 1 protocol is described in detail at
    https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt
 
+[[HiddenServiceExportRendPoint]] **HiddenServiceExportRendPoint** **0**|**1**::
+   When HiddenServiceExportCircuitID along with this feature is enabled,
+   export IP and port of the rendezvous point along with the circuit ID. +
```

Comment:
Why require HiddenServiceExportCircuitID to be enabled?

Reply:
Because that directive specifies the protocol(haproxy proxy). We could create another directive like `HiddenServiceExportProtocol` and make `HiddenServiceExportCircuitID` a bool, but that way we will introduce breaking changes.

@teor2345 @dgoulet @ahf opinions?

Reply:
It's not a particularly clean design, but I think it's ok as long as it's documented. If I was designing this from scratch, we'd have one option for the protocol, and then just export all the fields. (Or have one option per field.)